src/ucp/proto/proto_am.inl

```diff
@@ -152,25 +156,35 @@ void ucp_dt_iov_copy_uct(ucp_context_h context, uct_iov_t *iov, size_t *iovcnt,
         length_it = iov[0].length;
         break;
     case UCP_DATATYPE_IOV:
-        iov_offset                  = state->dt.iov.iov_offset;
-        max_src_iov                 = state->dt.iov.iovcnt;
-        src_it                      = state->dt.iov.iovcnt_offset;
-        dst_it                      = 0;
-        state->dt.iov.iov_offset    = 0;
+        iov_offset               = state->dt.iov.iov_offset;
```

Comment:
the iov code already became too big, can you export to separate function?

Reply:
done